<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>286</width>
    <height>500</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>500</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="locale">
   <locale language="English" country="Germany"/>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>480</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>480</height>
      </size>
     </property>
     <property name="locale">
      <locale language="English" country="Germany"/>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <property name="fieldGrowthPolicy">
       <enum>QFormLayout::AllNonFixedFieldsGrow</enum>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="label_7">
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="QLineEdit" name="editDescription">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="label_20">
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="text">
         <string>Distance [m]</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="editDistance">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="maxLength">
         <number>7</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_8">
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="text">
         <string>Speed [m/s]</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QLineEdit" name="editSpeed">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="maxLength">
         <number>4</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_19">
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="text">
         <string>Time [s]</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1">
       <widget class="QLineEdit" name="editTime">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="maxLength">
         <number>10</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_9">
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="text">
         <string>Timeout [s]</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QLineEdit" name="editTimeout">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="maxLength">
         <number>11</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QLabel" name="label_21">
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="text">
         <string>Supress Propulsion</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1">
       <widget class="QComboBox" name="comboSupressPropulsion">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <item>
         <property name="text">
          <string>False</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>True</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QLabel" name="label_10">
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="text">
         <string>Priority</string>
        </property>
       </widget>
      </item>
      <item row="6" column="1">
       <widget class="QSpinBox" name="spinBoxPriority">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QLabel" name="label_12">
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="text">
         <string>Pitch Control</string>
        </property>
       </widget>
      </item>
      <item row="7" column="1">
       <widget class="QComboBox" name="comboPitchControl">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <item>
         <property name="text">
          <string>No</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Yes</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="8" column="0">
       <widget class="QLabel" name="label_13">
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="text">
         <string>Pitch SetPoint</string>
        </property>
       </widget>
      </item>
      <item row="8" column="1">
       <widget class="QLineEdit" name="editPitchSetPoint">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="9" column="0">
       <widget class="QLabel" name="label_14">
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="text">
         <string>Arrival Radius [m]</string>
        </property>
       </widget>
      </item>
      <item row="9" column="1">
       <widget class="QLineEdit" name="editArrivalRadius">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="maxLength">
         <number>5</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="10" column="0">
       <widget class="QLabel" name="label_15">
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="text">
         <string>Look Ahead Distance [m]</string>
        </property>
       </widget>
      </item>
      <item row="10" column="1">
       <widget class="QLineEdit" name="editLookAheadDistance">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="11" column="0">
       <widget class="QLabel" name="label_16">
        <property name="toolTip">
         <string/>
        </property>
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="text">
         <string>Distance to LOS [m]</string>
        </property>
       </widget>
      </item>
      <item row="11" column="1">
       <widget class="QLineEdit" name="editDistToLOS">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="maxLength">
         <number>7</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="12" column="0">
       <widget class="QLabel" name="label">
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="text">
         <string>Track Controller Mode</string>
        </property>
       </widget>
      </item>
      <item row="12" column="1">
       <widget class="QComboBox" name="comboTrackControllerMode">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <item>
         <property name="text">
          <string>CTE</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>LOS</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="13" column="0">
       <widget class="QLabel" name="label_6">
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="text">
         <string>Depth Controlle rMode</string>
        </property>
       </widget>
      </item>
      <item row="13" column="1">
       <widget class="QComboBox" name="comboDepthControllerMode">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <item>
         <property name="text">
          <string>ConstantDepth</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>HeightOverGround</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="14" column="0">
       <widget class="QLabel" name="label_17">
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="text">
         <string>Invalid Height Iterations</string>
        </property>
       </widget>
      </item>
      <item row="14" column="1">
       <widget class="QLineEdit" name="editHeightIterations">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="15" column="0">
       <widget class="QLabel" name="label_18">
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="text">
         <string>Depth If Heigth Invalid [m]</string>
        </property>
       </widget>
      </item>
      <item row="15" column="1">
       <widget class="QLineEdit" name="editDepthIfHeightInvalid">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="16" column="0">
       <widget class="QLabel" name="label_4">
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="text">
         <string>Height Over Ground [m]</string>
        </property>
       </widget>
      </item>
      <item row="16" column="1">
       <widget class="QLineEdit" name="editHeightOverGround">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="17" column="0">
       <widget class="QLabel" name="label_11">
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="text">
         <string>Constant Depth Value</string>
        </property>
       </widget>
      </item>
      <item row="17" column="1">
       <widget class="QLineEdit" name="editConstantDepth">
        <property name="maximumSize">
         <size>
          <width>150</width>
          <height>16777215</height>
         </size>
        </property>
        <property name="locale">
         <locale language="English" country="Germany"/>
        </property>
        <property name="maxLength">
         <number>8</number>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
